### PR TITLE
feat(app-icon): Upgrade CozyUI+CozyClient+MUI to display app icon faster

### PR DESCRIPTION
**Pull request by Ldoppea**
* Loading App's icons: Should be faster. We now rely on img src=http directly if we are not in OAuth. Otherwise we do a fetch.

* Maintenance task to upgrade cozy-ui and cozy-client to latest versions

I also homogenised icons usage by using icons as components (vs string literals) everywhere when possible

### CODE REVIEW

File: src/ducks/apps/components/Sections/components/DropdownFilter.jsx [hunk 1]
```diff
@@ -2,12 +2,13 @@ import React, { Component } from 'react'
...
 import SelectBox from 'cozy-ui/transpiled/react/SelectBox'
 import Icon from 'cozy-ui/transpiled/react/Icon'
+import Bottom from 'cozy-ui/transpiled/react/Icons/Bottom'
 import PropTypes from 'prop-types'
 
 const SmallArrow = () => (
   <Icon
     className="sto-sections-dropdown-icon"
-    icon="bottom"
+    icon={Bottom}
```
Comment (trollepierre) [inline src/ducks/apps/components/Sections/components/DropdownFilter.jsx:11]: this change is not tested, can we try adding one test? 🙏

File: src/ducks/search/components.jsx [hunk 2]
```diff
@@ -34,7 +35,7 @@ export const SearchField = ({ onChange, value }) => {
         className={isMobile ? '' : 'u-mb-1'}
         prepend={
           isMobile ? (
-            <Icon icon="magnifier" className="u-pl-1 u-coolGrey" />
+            <Icon icon={Magnifier} className="u-pl-1 u-coolGrey" />
```
Comment (trollepierre) [inline src/ducks/search/components.jsx:38]: this change is not tested, can we try adding one test? 🙏

File: test/ducks/apps/components/__snapshots__/uninstallModal.spec.js.snap [hunk 3]
```diff
@@ -143,7 +143,7 @@ exports[`UninstallModal component should handle correctly linked app error 1`] =
         className="u-mh-half"
         disabled={true}
         extension="full"
-        icon="trash"
+        icon={[Function]}
```
Comment (trollepierre) [inline test/ducks/apps/components/__snapshots__/uninstallModal.spec.js.snap:146]: I recommend mocking the Trash for this snap and verify trash is present in the snap

File: test/ducks/apps/components/applicationPage/__snapshots__/index.spec.js.snap [hunk 4]
```diff
@@ -21,7 +21,7 @@ exports[`ApplicationPage component should be rendered correctly with app infos f
     >
       <DefaultButton
         className="sto-app-back"
-        icon="left"
+        icon={[Function]}
```
Comment (trollepierre) [inline test/ducks/apps/components/applicationPage/__snapshots__/index.spec.js.snap:24]: I recommend mocking the Left for this snap and verify Left mock is present in the snap

**Review by trollepierre — CHANGES_REQUESTED** [review]

File: src/ducks/apps/components/Sections/components/DropdownFilter.jsx (hunk 1 above)
Comment (Ldoppea) [reply src/ducks/apps/components/Sections/components/DropdownFilter.jsx:11]: I'm not sure why we want to test a graphic item 🤔 

Or maybe what you want is a snapshot test?

**Review by trollepierre — APPROVED** [review]

**Pull request merged**